README.rst

```diff
@@ -63,6 +87,12 @@ PostgreSQL instance, (make sure you installed `podman-compose
   $ # Also available from PyPI: pip install podman-compose
   $ podman-compose up
 
+Then, sync your local instance using ``pgsql``:
```

Comment:
maybe better lower before `If you would like to experiment with PostgreSQL programmatically,...`